x-pack/plugins/security/public/views/overwritten_session/_overwritten_session.scss

```diff
@@ -0,0 +1,25 @@
+.overwrittenSession {
```

Comment:
__note:__ proudly copied from `logged_out` view :)

Reply:
Is there **ANY** way we can start making this pattern a re-usable component instead of copy/pasting? It's getting hard to manage all the usages to make sure updates get applied everywhere.

Reply:
We can try, let me check how many times this one is actually copy-pasted. Do you have any re-usable component in mind we can use as an example here?

Reply:
The latest one we tackled was the help menu popover that had default content but also allowed for custom content https://github.com/elastic/kibana/blob/master/src/legacy/ui/public/chrome/directives/header_global_nav/components/header_help_menu.tsx.

I'd assume the customizable bits would be the title string and the main content.